src/devtools/devtools.tsx

```diff
@@ -204,13 +224,13 @@ export function ReactQueryDevtools({
     <Container ref={rootRef} className="ReactQueryDevtools">
       <ThemeProvider theme={theme}>
         <ReactQueryDevtoolsPanel
-          ref={panelRef}
+          ref={panelRef as any}
```

Comment:
what error would you be getting here please?

Reply:
I added the error I was facing in this comment -
https://github.com/tannerlinsley/react-query/pull/2688#discussion_r711621930